migrations/20240924045045_access_request.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS af_access_request (
+  request_id UUID NOT NULL DEFAULT gen_random_uuid(),
+  uid BIGINT NOT NULL REFERENCES af_user(uid) ON DELETE CASCADE,
+  workspace_id UUID NOT NULL,
+  view_id UUID NOT NULL,
+  approved BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
imo we should have at least 3 states for status, pending/approve/rejected (default pending). in that way when access is rejected, it is easier to query for the rest of pending access request